### PR TITLE
Creating Product controller

### PR DESCRIPTION
**Pull request by LesliNieto**
In this branch I worked on the product controller and the view of product list.

### CODE REVIEW

File: app/assets/javascripts/products.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Jucerweb) [inline app/assets/javascripts/products.coffee]: Delete all these empty files

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root 'products#index'
+  get '/products' , to:'products#index'
```
Comment (Jucerweb) [inline config/routes.rb]: Change this method for `resources`

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'products#index'
+  resources :products
```
Comment (Jucerweb) [inline config/routes.rb]: only the route you need in this moment.

Comment (LesliNieto) [reply config/routes.rb]: Ok

**Pull request merged**